addons/website_sale/static/src/js/website_sale_configurators.js

```diff
@@ -90,7 +90,10 @@ WebsiteSale.include({
             soDate: serializeDateTime(DateTime.now()),
             edit: false,
             isFrontend: true,
-            options: { isMainProductConfigurable: !isOnProductPage },
+            options: {
+                isMainProductConfigurable: !isOnProductPage,
+                showQuantity: Boolean(this.$form[0].querySelector('.show_quantity')),
```

Comment:
I wonder whether we can simply rely on `css_quantity` instead of adding a new class. Not sure though, maybe we need a more specific selector.